index.html

```diff
@@ -0,0 +1,166 @@
+<!DOCTYPE html>
+<html lang="es">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link
+      rel="stylesheet"
+      href="https://fonts.googleapis.com/css?family=Lato"
+    />
+    <link rel="stylesheet" href="https://fonts.cdnfonts.com/css/cocogoose" />
+    <link rel="stylesheet" href="resources/css/styles.min.css" />
+    <title>Expo tuiteratura</title>
+  </head>
+  <body>
+    <div class="wrapper">
+      <header>
+        <nav>
+          <div class="menu">
+            <div class="menu-top">
+              <ul class="menu-top-list">
+                <li class="menu-top-item">
+                  <a href="#"
+                    ><img src="resources/images/icon_facebook.png" alt=""
+                  /></a>
+                </li>
+                <li class="menu-top-item">
+                  <a href="#"
+                    ><img src="resources/images/icon_twitter.png" alt=""
+                  /></a>
+                </li>
+                <li class="menu-top-item">
+                  <a href="#">English</a>
+                </li>
+              </ul>
+            </div>
+            <div class="menu-main">
+              <div class="menu-img">
+                <img
+                  class="menu-button"
+                  src="resources/images/icon-menu.svg"
+                  alt="menu icon"
+                />
+                <img
+                  class="menu-logo"
+                  src="resources/images/logo.png"
+                  alt="logo"
+                />
+                <div>&nbsp;</div>
+              </div>
+              <ul id="menu-list" class="menu-list hidden">
+                <li class="menu-item"><a href="index.html">Home</a></li>
+                <li class="menu-item"><a href="about.html">About</a></li>
+                <li class="menu-item">
+                  <a href="mailto:sonaban.art@gmail.com">Contact</a>
+                </li>
+              </ul>
```

Comment:
- [ ] Great job so far, the navbar is missing some elements from the design, to stick to the design please include every element presented in the design and also there is no Home link in the design please remove that.

### Behance Design

![image](https://user-images.githubusercontent.com/59416136/163451849-79207763-7de2-407e-b859-9f7f29334606.png)

Reply:
@Wes-Isaac Thank you for your feedback. I'm working on the changes. I have a question. Since there's no home link, where it should be? Because the About page has the same menu and I suppose I need to go back to the home page.

Reply:
Hi @rbanos-mv I understand for now we can use the back button the browser provides. So that we can stick to the design